src/app/content/components/Attribution.spec.tsx

```diff
@@ -179,21 +179,15 @@ describe('Attribution', () => {
     });
 
     it('renders special licensing and attribution for HS Physics', async() => {
-
-      const { node } = renderToDom(render());
-      const details = node;
-
-      if (!document) {
-        return expect(document).toBeTruthy();
-      }
-
-      details.setAttribute('open', '');
-
       store.dispatch(
         actions.receiveBook({...formatBookData(book, mockCmsBook), id: 'cce64fde-f448-43b8-ae88-27705cceb0da'})
       );
+      const { node: details } = renderToDom(render());
+      details.setAttribute('open', '');
+
+      const matchTEAText = 'you must attribute “Texas Education Agency (TEA)."';
```

Comment:
You could also render this with `react-test-renderer` and then check for `<FormattedHTMLMessage>` with id `'i18n:attribution:tea-text'` and specifc values

^^ This is a non-blocking comment